Block/Compatibility.php

```diff
@@ -0,0 +1,76 @@
+<?php /** @noinspection PhpUnused */
```

Comment:
Which one is unsued? I can't see from the code

Reply:
Removed